KarhooUISDK/CommonUI/BottomSheet/KarhooBottomSheet.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  SwiftUIView.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 03.11.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct KarhooBottomSheet<Content: View>: View {
+    // MARK: - Properties
+    private var viewModel: BottomSheetViewModel
+    @ViewBuilder private let content: () -> Content
+    @ObservedObject private var keyboard = KeyboardResponder()
+    @State private var offset = CGSize.zero
+
+    // MARK: - Init
+    init(viewModel: BottomSheetViewModel, @ViewBuilder content: @escaping () -> Content) {
+        self.viewModel = viewModel
+        self.content = content
+    }
+   
+    // MARK: - Body
+    var body: some View {
+        VStack(spacing: 0.0, content: {
+            Spacer()
+                .background(Color.clear)
+                .layoutPriority(Double(UILayoutPriority.defaultLow.rawValue))
+            
+            VStack(spacing: 0.0) {
+                HStack {
+                    Text(viewModel.title)
+                        .font(Font(KarhooUI.fonts.bodyBold()))
+                        .foregroundColor(KarhooUI.colors.text.getColor())
+                    Spacer()
+                        .background(Color.clear)
+                    Button {
+                        viewModel.dismiss()
+                    } label: {
+                        Image(
```

Comment:
Missing circled background for cross icon
<img width="1162" alt="Screenshot 2022-11-17 at 23 55 02" src="https://user-images.githubusercontent.com/97114085/202577544-eed708dd-74e8-4c49-a7c7-5748c5ef6fe9.png">

Reply:
We decided at some point to use the same X icon we were already using for sort by and filters. We have the same icon on Android as on iOS at the moment (for the master bottom sheet).